### ISSUE TITLE
Creating a support/discussion forum for BRT PG online

### ISSUE THREAD

**Issue by dcpaschall** [ITDP/the-online-brt-planning-guide#104]
Is there a way that we can create a forum for users of the guide to discuss issues having to do with the guide, such as a google group or some other type of platform? One example is the google group here for the Open Streets Project: http://openstreetsproject.org/

**Comment by jonasmalacofilho** [comment]
Yeah, that's a good idea.  But are you thinking about issues with the online guide "mechanics" (such as the syntax or git) or about the guide itself?

Besides a good mailing list/user group (like Google Groups) we could also consider Gitter (a chat platform that integrates with GitHub and has channels and history) or even Slack (a more general project/org communication platform), depending on the subject and audience.

**Comment by jonasmalacofilho** [comment]
Oh, and for some things "question" issues here are great too...

**Comment by aegauthier** [comment]
I am thinking both - mechanics and the guide. So that if people are having mechanical problems, at least there is place for them to ask about it and then we can redirect them to better resources. I would want something that archives the discussions so people can refer to them later. First stop with a question is to see if it has been asked before and what the answer is. Then they can query the larger group if not there. Google groups does that fine. Again, the simplest way would be best - so people don't have to learn more new things or download more applications is my preference - so google groups is where I am leaning.

**Comment by aszasz** [comment]
@dcpaschall, @aegauthier: Go ahead and make the user group with the tool you think is easier to use and send us invitations (maybe assign that  to Thais, so she becomes the manager).... I think Google Groups is good enough...

...but, just to mention, learning Slack is easy (the barrier is set it up as a fix tab on your browser... easier than Hangouts, which we all learned recently despite our initial resistance... definitely easier than git), you might investigate if there is a way to have it open to anyone who wants to join, without an invitation... the advantage is that it is a chat and we can be online.

**Comment by jonasmalacofilho** [comment]
I'm fine with a Google Group as well.

Just a thought, but...

> Again, the simplest way would be best - so people don't have to learn more new things or download more applications

Considering we would need to point the users to whatever we choose for the support "forum", how about using GitHub issues for this?  Could they figure out the the issue system, once pointed directly here (as planned in #55)?

**Comment by jonasmalacofilho** [comment]
@thaiscmlima do you have any ideia about the status of this?
